stingray/pulse/search.py

```diff
@@ -164,17 +174,18 @@ def z_n_search(times, frequencies, nharm=4, nbin=128, segment_size=5000,
         if expocorr and gti is None:
             raise ValueError('To calculate exposure correction, you need to'
                              ' specify the GTIs')
-        def fun(t, f, **kwargs):
+        def fun(t, f, fd=0, **kwargs):
```

Comment:
"fun" is not a super descriptive name for a function. :)

Reply:
Done :)